src/lib/components/MediaObject/MediaObject.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './MediaObject.css';
+
+const MediaObject = (props) => {
+  let image;
+  let title;
+  let description;
+  let metadata;
+
+  if (props.img) {
+    image = (
+      <img
+        className={`p-media-object__image${props.round ? ' is-round' : ''}`}
+        src={props.img.src}
+        alt={props.img.alt}
+      />
+    );
+  }
+
+  if (props.title) {
+    if (props.title.link) {
+      title = (
+        <h3 className="p-media-object__title">
+          <a href={props.title.link}>{props.title.name}</a>
+        </h3>
+      );
+    } else {
+      title = (
+        <h3 className="p-media-object__title">
+          {props.title.name}
+        </h3>
+      );
+    }
+  }
+
+  if (props.description) {
+    description = (
+      <p className="p-media-object__content">
+        {props.description}
+      </p>
+    );
+  }
+
+  if (props.metadata) {
+    const metadataItems = Object.keys(props.metadata).map((key) => {
+      if (key === 'topic') {
+        return (
+          <li key={key} className="p-media-object__meta-list-item">
+            {props.metadata[key]}
+          </li>
+        );
+      }
+      return (
+        <li key={key} className={`p-media-object__meta-list-item--${key}`}>
+          {props.metadata[key]}
+        </li>
+      );
+    });
+
+    metadata = (
+      <ul className="p-media-object__meta-list">
+        {metadataItems}
+      </ul>
+    );
+  }
+
+  return (
+    <div className={props.large ? 'p-media-object--large' : 'p-media-object'}>
+      {image}
+      <div className="p-media-object__details">
+        {title}
+        {description}
+        {metadata}
+      </div>
+    </div>
+  );
+};
+
+MediaObject.defaultProps = {
+  round: false,
+  large: false,
+  img: null,
```

Comment:
is it really usefull to put defaultProps when it's `null`?

Reply:
Not really. It's basically there to satisfy the [current linting rule](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/require-default-props.md) that every prop that is not required MUST have a default value. We could revisit the current linting rules in the future, but for now this option allows people to use the MediaObject component without explicitly declaring what props they don't want to include - only the ones they do want.

Reply:
ok then it's good to me 👍